src/Indicator.vala

```diff
@@ -130,10 +135,10 @@ public class Session.Indicator : Wingpanel.Indicator {
 
                 main_grid.add (scrolled_box);
                 main_grid.add (user_settings);
-                main_grid.add (users_separator);
+                main_grid.add (new Gtk.Separator (Gtk.Orientation.HORIZONTAL));
                 main_grid.add (lock_screen);
                 main_grid.add (log_out);
-                main_grid.add (new Wingpanel.Widgets.Separator ());
+                main_grid.add (new Gtk.Separator (Gtk.Orientation.HORIZONTAL));
```

Comment:
Just wondering if it's OK to replace `Wingpanel.Widgets.Separator` with `Gtk.Separator`―small thing but keep the UI the same I guess we may need to add 3px margins at the top and bottom of each `Gtk.Separator` here, as per https://github.com/elementary/wingpanel/blob/b71e65a20bf183e58f47e54bebce874898c8dd9a/lib/Widgets/Separator.vala#L24-L25